test/nn/test_convolution.py

```diff
@@ -2054,7 +2045,8 @@ def test_ConvTranspose3d_size_1_kernel(self, device):
 
             self.assertEqual(y_cpu, y_cuda, atol=1e-5, rtol=0, exact_device=False)
             self.assertEqual(conv_cpu.bias.grad.data, conv_cuda.bias.grad.data, atol=1e-5, rtol=0, exact_device=False)
-            self.assertEqual(conv_cpu.weight.grad.data, conv_cuda.weight.grad.data, atol=1e-5, rtol=0, exact_device=False)
+            self.assertEqual(conv_cpu.weight.grad.data, conv_cuda.weight.grad.data,
```

Comment:
Is this and everything above just formatting change?
For future PR, please do these in a separate PR to ease review.

Reply:
I'm sorry, I usually do. I must have let this slip.